app/main.py

```diff
@@ -1 +1,40 @@
-#  write your code here
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
```

Comment:
The `feed` method in the `Animal` class uses an `if` statement to check if the animal is hungry, and if it's not, it returns 0. However, there's no need for the `else` clause. You can simply return 0 at the end of the method. This way, if the animal is hungry, the method will return the appetite value and if not, it will return 0. This is a simpler and cleaner way to write this method and it follows the principle of making code easier to read and understand. This is related to the checklist point 'Don't use an extra else statement'. Here's how you can refactor this method: python def feed(self) -> int: if self.is_hungry: print(f"Eating {self.appetite} food points...") self.is_hungry = False return self.appetite return 0 ```